package.json

```diff
@@ -37,6 +37,7 @@
     "ordinal": "^1.0.3",
     "parse": "^2.13.0",
     "prop-types": "^15.7.2",
+    "randomstring": "^1.1.5",
```

Comment:
take the randomString() function from rwallet-server.

Let's not add another npm package for such a small utility function

Reply:
I replace it